src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -146,6 +112,53 @@ bool same_host_mem(cldnn::memory::ptr memPtr, uint8_t* hostPtr) {
 namespace ov {
 namespace intel_gpu {
 
+template<typename src_dt, typename dst_dt>
+void InferRequest::copyResultToOutputBlob(cldnn::memory::ptr src, InferenceEngine::Blob::Ptr dst, bool need_convert) {
```

Comment:
template member functions should be defined in header or have explicit instantiations in cpp file to avoid linkage issues in some cases

Reply:
Fixed, thanks